bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -755,7 +761,13 @@ impl ParseError {
         move |source| Self::InvalidInteger { source, input: s.into() }
     }
 
-    fn display(&self, f: &mut fmt::Formatter<'_>, subject: &str, lower_bound: u32, upper_bound: u32) -> fmt::Result {
+    fn display(
+        &self,
+        f: &mut fmt::Formatter<'_>,
+        subject: &str,
+        lower_bound: u32,
+        upper_bound: u32,
+    ) -> fmt::Result {
```

Comment:
In e6957d2259b34323e5bed7d33651164d3e8626ac:

This change is unrelated to the PR.

Reply:
> In e6957d2259b34323e5bed7d33651164d3e8626ac:
> 
Ah,I see now - that change comes from running  `Cargo fmt` .